frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -233,5 +234,21 @@ def rescue_from_spree_gateway_error(exception)
     def check_authorization
       authorize!(:edit, current_order, cookies.signed[:guest_token])
     end
+
+    def insufficient_stock_error
+      packages = @order.shipments.map(&:to_package)
+      if packages.empty?
+        flash[:error] = I18n.t('spree.insufficient_stock_for_order')
+        redirect_to cart_admin_order_url(@order)
```

Comment:
This path should probably not contain `_admin_`. Also maybe we should use `_path` instead of `_url`, right?

Reply:
good catch! 👍